tls/s2n_x509_validator.c

```diff
@@ -32,6 +32,8 @@
 
 #if !defined(OPENSSL_IS_BORINGSSL)
```

Comment:
Should this be using the same check you have in the `s2n_x509_validator_validate_cert_stapled_ocsp_response` function?

```suggestion
#if !defined(S2N_OCSP_STAPLING_SUPPORTED)
```

Reply:
Yes, that makes more sense. I fixed this in 83f9db8ab8d5ce464f22bf5ebe10159ff4b34fd0 and also updated the check in the test that calls this in 0cc0d4b398067aeb65e95af4b5d4d80de7f2a0ed.

Reply:
Fixed this further in d2b61d1e3988808370f56b2f9f1d01ddb74e8346.